src/SmartComponents/Recs/List.scss

```diff
@@ -1,10 +1,14 @@
 .ins-c-recommendations-header {
     white-space: nowrap;
     width: auto;
-    > * {
+    .pf-c-title {
         display:inline-block;
     }
     .pf-c-button {
         line-height: var(--pf-global--LineHeight--sm);
     }
+    .alertOverride {
+        font-size: var(--pf-global--FontSize--md);
+        margin-top: 24px;
```

Comment:
Hey @mhuth you could probably switch this px value for `margin-top: var(--pf-global--spacer--lg);`

Reply:
Done!  Good suggestion @LI0131 !